sycl/doc/EnvironmentVariables.md

```diff
@@ -38,8 +38,9 @@ subject to change. Do not rely on these variables in production code.
 | `SYCL_ENABLE_PCI` | Integer | When set to 1, enables obtaining the GPU PCI address when using the Level Zero backend. The default is 0. |
 | `SYCL_HOST_UNIFIED_MEMORY` | Integer | Enforce host unified memory support or lack of it for the execution graph builder. If set to 0, it is enforced as not supported by all devices. If set to 1, it is enforced as supported by all devices. |
 | `SYCL_CACHE_DIR` | Path | Path to persistent cache root directory. Default values are `%AppData%\libsycl_cache` for Windows and `$XDG_CACHE_HOME/libsycl_cache` on Linux, if `XDG_CACHE_HOME` is not set then `$HOME/.cache/libsycl_cache`. |
-| `SYCL_CACHE_TRACE` | Any(\*) | If the variable is set, messages are sent to std::cerr when non-blocking failures happen (e.g. unable to access cache item file). |
-| `SYCL_CACHE_DISABLE_PERSISTENT` | Any(\*) | Switches persistent cache switch off when the variable is set. |
+| `SYCL_CACHE_TRACE` | Any(\*) | If the variable is set, messages are sent to std::cerr when caching events or non-blocking failures happen (e.g. unable to access cache item file). |
+| `SYCL_CACHE_DISABLE_PERSISTENT (deprecated)` | Any(\*) | Has no effect. |
```

Comment:
Does it make sense to rework this var in similar to SYCL_HOST_UNIFIED_MEMORY, which is basically bool?
Call it SYCL_CACHE_PERSISTENT. Make it zero = OFF and non-zero = ON. Default can change independently based on what we decide now and later?
Looks like better control over behavior and more stable as a variable.

Reply:
addressed in 2964f91